Makefile

```diff
@@ -49,20 +49,30 @@ compile_go_mock_market_place:
 
 compile_mock_market_place: compile_solidity_mock_market_place compile_go_mock_market_place
 
-compile_mocks: compile_mock_erc compile_mock_yield_token compile_mock_market_place
+compile_solidity_mock_zc_token:
+	@echo "compiling Mock YToken solidity source into abi and bin files"
+	solc -o ./test/mocks --abi --bin --overwrite ./test/mocks/ZcToken.sol
+
+compile_go_mock_zc_token:
+	@echo "compiling abi and bin files to golang"
+	abigen --abi ./test/mocks/ZcToken.abi --bin ./test/mocks/ZcToken.bin -pkg mocks -type ZcToken -out ./test/mocks/zctoken.go 
+
+compile_mock_zc_token: compile_solidity_mock_zc_token compile_go_mock_zc_token
+
+compile_mocks: compile_mock_erc compile_mock_yield_token compile_mock_market_place compile_mock_zc_token
 
 # Real Tokens
-# compile_solidity_zct:
-	# @echo "compiling ZCT solidity source into abi and bin files"
-	# solc -o ./test/tokens --abi --bin --overwrite ./test/tokens/ZcToken.sol
+compile_solidity_zct:
```

Comment:
i don't think this is necessary. Swivel has an actual ZCT contract because the marketplace deploys one. Illuminate, I believe, will only ever be communicating with an already deployed ZCT (unless create market deploys one, which i'm not sure of off the top of my head. -- regardless it's not needed yet)